src/util/reports/campPhotos.js

```diff
@@ -60,9 +60,7 @@ module.exports = (startDate, endDate) =>
     .flatMap(sessions => assignDocumentData(sessions, streamRegistrations, 'sessionId', 'sessionId', 'registration'))
     .flatMap(({ registration, ...rest }) => hl(registration.registrationDetails || [])
       .filter(({ cancelled }) => !cancelled)
-      .map(({ tuitionId, personId, sessionOptions }) => Object.assign({ tuitionId, personId, sessionOptions }, rest)))
-    .collect()
-    .flatMap(assignHasPurchasedPhotos)
```

Comment:
why was this removed?

Reply:
SessionOptions are not be used for any field for excel creation, so i think it will be an extra request, so i remove it.